keyboards/xelus/valor/readme.md

```diff
@@ -0,0 +1,13 @@
+# Valor
+
+An Alice replacement keyboard PCB.
+
+* Keyboard Maintainer: [Xelus22](https://github.com/Xelus22)
+* Hardware Supported: CONSTELLATION
+* Hardware Availability: Custom keyboard group buys
+
+Make example for this keyboard (after setting up your build environment):
+
+    make aeboards/constellation:default
```

Comment:
`aeboards/constellation`?

Reply:
Sorry, I just copy pasted this.  Forgot to change it over.